src/Services/SettingsService.php

```diff
@@ -358,6 +398,30 @@ private function loadClientSettings($plentyId, $lang)
         return $clientSettings;
     }
 
+    /**
+     * Check if settings exist for plentyId and language
+     *
+     * @param $plentyId
+     * @param $lang
+     *
+     * @return boolean
+     */
+    public function clientSettingsExist($plentyId, $lang)
+    {
+        /** @var Query $query */
+        $query = $this->db->query(Settings::MODEL_NAMESPACE);
+        $query->where('plentyId', '=', $plentyId);
+        if(!empty($lang))
+        {
+            $query->where('lang', '=', $lang);
+        }
+        $query->orWhere('lang',   '=', '')->where('plentyId', '=', $plentyId);
```

Comment:
Warum fügst du die Where-Bedingung: `->orWhere('lang',   '=', '')` immer hinzu, selbst in dem Fall falls eine Sprache vorhanden ist...?

Reply:
Es gibt manche Optionen, die ohne eine Sprache gespeichert werden.
![Bildschirmfoto 2019-10-02 um 14 11 43](https://user-images.githubusercontent.com/50198131/66043151-b0d46580-e51e-11e9-833d-343a210183d5.png)